docs/loot/recipes.md

```diff
@@ -0,0 +1,625 @@
+			"identifier": "lichbane:cold_steel_sword"
+		},
+		"tags": ["crafting_table", "altar"],
+
+		"pattern": [
+			"X",
+			"X",
+			"I"
+		],
+		"key": {
+			"X": "lichbane:cold_steel",
+			"I": "minecraft:stick"
+		},
+		"result": "lichbane:cold_steel_sword"
+	}
+}
+```
+
+::: tip NOTE
+Anvil interactions are handled within an [item definition](https://wiki.bedrock.dev/items/items-16.html). Smithing table and loom transactions are currently unavailable.
+:::
+
+All recipes are stored in the `recipes` folder in the behavior pack root. The files can be named and organized under any folder hierarchy as desired. No experimental toggles are required to use recipes or any of their features.
+
+## Shared Properties and Structures
+### Format Version
+The format version is intended to version the schema used for the body of a recipe. It is provided with the top-level `"format_version"` property.
+
+<CodeHeader>#/</CodeHeader>
+```json
+"format_version": "1.17.41"
+```
+
+In practice, the format version can be set to any value or even omitted.
+
+::: warning
+It's strongly recommended to include a format version anyway, choosing a value that represents an actual Minecraft version to help future-proof the code. Consider using the current release version or last major release version.
+:::
+
+### Description
+The `"description"` object, required within any recipe type, holds the metadata of a recipe.
+
+<CodeHeader>#/minecraft:recipe_shaped/</CodeHeader>
+```json
+"description": {
+	"identifier": "lichbane:cold_steel_sword"
+}
+```
+
+Its only child is the required `"identifier"` property, which is designed to uniquely identify a recipe across all packs applied to a world. There are no namespacing requirements for recipe identifiers except that no two full recipe identifiers in a single pack may match.
+
+::: warning
+It's strongly recommended to use a namespace. Namespaces are a standard in other add-on domains and assist in logically scoping the recipe to a pack, lessening possibilities of collisions for players wanting to use multiple behavior packs in their world.
+:::
+
+### Tags
+Recipes are linked to crafting interfaces using the required `"tags"` array property, which must be placed within any recipe type. Any interface whose tag is provided in the array may use that recipe, if applicable. At least one tag must be provided.
+
+<CodeHeader>#/minecraft:recipe_shaped/</CodeHeader>
+```json
+"tags": ["crafting_table", "altar"]
+```
+
+Vanilla interfaces are exposed to tags for each set of recipe types.
+
+Crafting:
+- `crafting_table`
+- `stonecutter`
+
+Cooking and Smelting:
+- `furnace`
+- `blast_furnace`
+- `smoker`
+- `campfire`
+- `soul_campfire`
+
+Brewing
+- `brewing_stand`
+
+Additionally, [custom crafting blocks can declare a custom tag](https://wiki.bedrock.dev/blocks/blocks-16.html#minecraft-crafting-table) for crafting recipes to use. Custom cooking and smelting blocks and custom brewing stands are not currently available.
```

Comment:
Remove https://wiki.bedrock.dev. Links must be relative

Reply:
Oh, that's just embarrassing on my part.